### PR TITLE
CLI interface

### PR DESCRIPTION
**Pull request by pocketjoso**
From @dhoko's PR #16, solves issue #14.

What I've changed/added:
* Embed JSON data into the HTML file generated by the CLI, this way it works cross browser.
* Remove possibility of specifying `output` as second argument, and fixed broken `-o --output` option.
* Added `-b --browser` option, which auto launches a browser to view generated CSS specificity graph.
* Updated README, remove old index.html example, fix general typos and weird things.

### CODE REVIEW

**Comment by pocketjoso** [comment]
@dhoko Are you okay with these changes? Will wait 24hours before I merge.

**Comment by dhoko** [comment]
ok for me, it works.

I don't know if it's from me or not but it doesn't open my browser :/ It opens Geany (a text editor)

File: README.md [hunk 1]
```diff
@@ -6,44 +6,65 @@ Idea by [Harry Roberts](http://csswizardry.com/2014/10/the-specificity-graph/)
...
+## Usage
+`specifity-graph` can be used in different ways:
+* [CLI](#cli)
+* [Via JavaScript (as a Node Module)](#via-javascript)
+* [Online](http://jonassebastianohlsson.com/specificity-graph/)
 
-### Standalone
-If you’re not using npm, you can simply download `specificity-graph-standalone.js`.
+### CLI
+`./bin/specificity-graph <cssFile> [options]`
```
Comment (dhoko) [inline README.md]: We can install it with -g so we do not need ./bin.

Comment (pocketjoso) [inline README.md]: Yeah I'm always torn about this.. Sure, will change.

**Comment by pocketjoso** [comment]
> I don't know if it's from me or not but it doesn't open my browser :/ It opens Geany (a text editor)

Hmm, the documentation says:
> open a file or url in the user's preferred application

Would think your OS would open a URL in a browser... Mine opens in Chrome.

**Comment by pocketjoso** [comment]
@dhoko Switched out the `open` npm module for [opn](https://github.com/sindresorhus/opn), should fix the issue of it opening in the wrong application. Can you confirm?

**Comment by dhoko** [comment]
Nope :/ 
But it's bugbuntu so I don't care^^

**Comment by pocketjoso** [comment]
Okay, I'll merge it then and it's something that can be improved on later.

**Comment by dhoko** [comment]
Ok it works on another computer :+1:

**Pull request merged**